src/Features/Core/Portable/GenerateDefaultConstructors/AbstractGenerateDefaultConstructorsService.State.cs

```diff
@@ -77,6 +84,26 @@ private State()
                 return UnimplementedConstructors.Length > 0;
             }
 
+            private static bool FixesError(INamedTypeSymbol classType, INamedTypeSymbol baseType)
+            {
+                // See if the user didn't supply a constructor, and thus the compiler automatically generated
+                // one for them.   If so, also see if there's an accessible no-arg contructor in the base.
+                // If not, then the compiler will error and we want the code-fix to take over solving this problem.
+                if (classType.Constructors.Any(c => c.Parameters.Length == 0 && c.IsImplicitlyDeclared))
+                {
+                    var baseNoArgConstructor = baseType.Constructors.FirstOrDefault(c => c.Parameters.Length == 0);
+                    if (baseNoArgConstructor == null ||
+                        !baseNoArgConstructor.IsAccessibleWithin(classType))
+                    {
+                        // this code is in error, but we're the refactoring codepath.  Offer nothing
+                        // and let the code fix provider handle it instead.
```

Comment:
This comment seems misplaced: "We're the refactoring codepath" - no you're not. This also gets called for the fix.

Reply:
will fixup